wgpu-core/src/command/transfer.rs

```diff
@@ -90,6 +96,18 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
         // borrow the buffer tracker mutably...
         let mut barriers = Vec::new();
 
+        #[cfg(feature = "trace")]
+        match cmb.commands {
```

Comment:
Looks like we could have `if let Some(ref mut list) = cmb.commands) { /* ... */ }` for these and some other places